### ISSUE TITLE
Bad alloc in hexedit

### ISSUE THREAD

**Issue by chivay** [codilime/veles#414]
Version: 1ae037156a0f6074001ef8fb9006a124596900bb (master)
```
#0  0x00007f91c8ccd860 in raise () from /usr/lib/libc.so.6
#1  0x00007f91c8cceec9 in abort () from /usr/lib/libc.so.6
#2  0x00007f91c9641d57 in __gnu_cxx::__verbose_terminate_handler () at /build/gcc/src/gcc/libstdc++-v3/libsupc++/vterminate.cc:95
#3  0x00007f91c963f8c6 in __cxxabiv1::__terminate (handler=<optimized out>) at /build/gcc/src/gcc/libstdc++-v3/libsupc++/eh_terminate.cc:47
#4  0x00007f91c963f913 in std::terminate () at /build/gcc/src/gcc/libstdc++-v3/libsupc++/eh_terminate.cc:57
#5  0x00007f91c963fb68 in __cxxabiv1::__cxa_throw (obj=<optimized out>, tinfo=0x7f91c992bdd0 <typeinfo for std::bad_alloc>, dest=0x7f91c963db60 <std::bad_alloc::~bad_alloc()>) at /build/gcc/src/gcc/libstdc++-v3/libsupc++/eh_throw.cc:93
#6  0x00007f91c96400cf in operator new (sz=0xffffffffffffffff) at /build/gcc/src/gcc/libstdc++-v3/libsupc++/new_op.cc:54
#7  0x000055e3cbb3c987 in veles::data::BinData::BinData (this=0x7ffc1461e6f0, width=0x8, size=0xffffffffffffffff, init_data=0x0) at ../include/data/bindata.h:50
50	      data_ = new uint8_t[octets()];
```

Steps to reproduce:
1. Open a non empty file
2. Select all (Ctrl + A)
3. Delete contents (Delete)
4. Try to delete contents once again (Delete)
5. Get a coredump

**Comment by mkow** [comment]
Could you please verify whether this issue was fixed by #415? (it's already on master)

**Comment by chivay** [comment]
OK, now it's working correctly.
